SentenceVectors.ecl

```diff
@@ -0,0 +1,651 @@
+  * - Collections of words can be analyzed to find outliers using FindLeastSimilar().
+  * <p>Sentence Vectors can be configured to incorporate word order and multi-word concepts using
+  * the wordNGrams feature.  When set to one, only individual words are considered, while when set
+  * to 3, one, two, and three word phrases are all considered words.  In this way a phrase like
+  * "traffic light" can have a very different meaning than either "traffic" or "light", and can also
+  * have a very different meaning from "light traffic".  "New York Times" can have a different meaning
+  * than "new", "york" or "times", as well as a different meaning than "New York" or "York Times".
+  * With wordNGrams set to 1, compound words as well as word order cannot be distinguished.  This can
+  * be useful for corpora that contain very terse cryptic sentences which may be equally valid in any
+  * order, and without compound word usage.
+  *
+  * <p>Sentence Vectors are trained using a specialized Neural Network that is task-optimized.  Batch
+  * Synchronous Stochastic Gradient Descent is used to parallelize the neural network training across
+  * the HPCC Cluster.
+  *
+  * @param vecLen The length in bytes of the sentence and word vectors to be learned (default 100).
+  * @param trainToLoss Stop training when the average Loss level reaches this level.
+  *        This is the recommended method of controlling training duration. The Default .05
+  *        (i.e. 5 percent loss) seems to give
+  *        good results.
+  * @param numEpochs The maximum number of times to loop through the full set of training data. Each
+  *                  pass through the training data is known as an Epoch.  Default 0 (recommended)
+  *                  means no fixed number of epochs.  Will use trainToLoss and noProgressEpochs
+  *                  to terminate.
+  *                  Note that the larger the corpus, the lower this
+  *                  value can become.  For an extremely large corpus, a single Epoch may be
+  *                  enough.
+  * @param batchSize The number of word pairs to process before synchronizing weights across
+  *                   the nodes during Stochastic Gradient Descent (SGD) (Default 0 -- auto assign --
+  *                   recommended).  Note that
+  *                   larger values imply longer periods between synchronization of weights across
+  *                   nodes.  Larger values also speed up the training as less time is spent
+  *                   synchronizing.  If the network fails to converge, this may be set too high.
+  * @param negSamples The number of random negative samples to generate for each positive sample
+  *                    (default 10).  This may be reduced to as little as 2 for an extremely large
+  *                    corpus.
+  * @param learningRate The rate at which the neural network learns (default .1).
```

Comment:
Default value of learningRate is .2 in the below code.

Reply:
Changed default to .1